test/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailerPreview < ActionMailer::Preview
+
+  def account_activation
+    user = User.find(101)
```

Comment:
Hehe, yeah that is the one account that I created myself with my own name and real email address. I'm also the admin. I will attempt to use a generic account later on for the preview.